test/controllers/DateOfBirthYesNoControllerSpec.scala

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import base.SpecBase
+import forms.YesNoFormProvider
+import models.{Name, NormalMode, OtherIndividual}
+import navigation.Navigator
+import org.mockito.Matchers.any
+import org.mockito.Mockito.when
+import org.scalatestplus.mockito.MockitoSugar
+import pages.individual.{DateOfBirthYesNoPage, NamePage}
+import play.api.inject.bind
+import play.api.test.FakeRequest
+import play.api.test.Helpers._
+import repositories.PlaybackRepository
+import views.html.DateOfBirthYesNoView
+
+import scala.concurrent.Future
+
+class DateOfBirthYesNoControllerSpec extends SpecBase with MockitoSugar {
+
+  val formProvider = new YesNoFormProvider()
+  val form = formProvider.withPrefix("otherIndividual.dateOfBirthYesNo")
+
+  lazy val dateOfBirthYesNoRoute = routes.DateOfBirthYesNoController.onPageLoad(NormalMode).url
+
+  val name = Name("New", None, "Protector")
```

Comment:
Just do a Ctrl+Shift+F on protector and make sure you're referring to other individual instead of protectors